### PR TITLE
Feature/664

### PR DESCRIPTION
**Pull request by jrmartin**
Fix graph resizing bug, #664

### CODE REVIEW

**Comment by ddelpiano** [comment]
@jrmartin I think we need to re-think the approach for this solution, I can see one conceptual issue that is the fact that we are using redux to propagate a change for a child component but this action does not really alterate the state, and since this is the case we don't really need to use redux.
The second issue is a tangible one and it's kind of similar to what we experienced with the other bug you fixed where the layer component was behind by 1 instance in term of updates, this might happen again here if multiple actions are fired at the same time since we can easily lose the resize action as bulked with something else that will fire the event.

An alternative solution that I can see is that we pass the height and width that we get in the [VFBMain](https://github.com/VirtualFlyBrain/geppetto-vfb/blob/027651a90e0e11ca9b6cfd9c840083a56e54ecbd/components/VFBMain.js#L1000) to the child component VFBGraph, inside the component we compute these values and check if they changed so that we can establish if the component needs to update itself or not.

Another viable solution is to use the onModelChange method provided by Flexlayout and do some computation there.

These are just the 2 that I thought looking at the code during the review and came quite immediately with the data available there  but if you want to tackle this with a different solution feel free.

One note, when I tried to merge this into development + feature/660 it caused me some conflicts, probably it's just faster to re-fork this branch from development but up to you.

Thanks!

**Review by ddelpiano — CHANGES_REQUESTED** [review]
changes documented in the comment left previously.

**Comment by jrmartin** [comment]
@ddelpiano I changed the approach, it's not longer using redux to resize the graph component. 
Instead, VFBMain is listening to FlexLayout node's resize events and resizes the graph and circuit browser after that. We don't need to re-calculate the width/height, since the react-force-graph-2d does a good job of resizing itself on re-rendering. 

This also resizes the circuit browser component.

File: components/interface/VFBCircuitBrowser/VFBCircuitBrowser.js [hunk 1]
```diff
@@ -428,6 +438,7 @@ class VFBCircuitBrowser extends Component {
               zoomIn={self.zoomIn}
               zoomOut={self.zoomOut}
               circuitQuerySelected={this.circuitQuerySelected}
+              datasource="SOLR"
```
Comment (jrmartin) [inline components/interface/VFBCircuitBrowser/VFBCircuitBrowser.js:441]: @ddelpiano I added this to this branch to resolve the conflicts between this branch and feature/660.

**Review by ddelpiano — APPROVED** [review]

**Pull request merged**